test/github-issues/108/issue-108.ts

```diff
@@ -0,0 +1,29 @@
+import "reflect-metadata";
+import { createTestingConnections, closeTestingConnections, reloadTestingDatabases } from "../../utils/test-utils";
+import { Connection } from "../../../src/connection/Connection";
+import { Project } from "./entity/Project";
+import { User } from "./entity/User";
+import { expect } from "chai";
+
+describe("github issues > #108 Error with constraint names on postgres", () => {
+
+    let connections: Connection[];
+    before(async () => connections = await createTestingConnections({
+        entities: [__dirname + "/entity/*{.js,.ts}"],
+        schemaCreate: true,
+        dropSchemaOnConnection: true,
+        enabledDrivers: ["postgres", "mssql"] // TODO:Add tests for mssql
+    }));
+    after(() => closeTestingConnections(connections));
+
+    it("should sync and create column names in the form uk_{tableName}_{columnName}", () => Promise.all(connections.map(async connection => {
+        await connection.syncSchema(true);
```

Comment:
because you have `schemaCreate: true, dropSchemaOnConnection: true,` connection settings sync schema already happens without this line of code